swagger/azure-parameter-grouping.json

```diff
@@ -5,9 +5,9 @@
     "description": "Test Infrastructure for AutoRest",
     "version": "1.0.0"
   },
-  "host": "localhost",
+  "host": "localhost:3000",

```

Comment:
port not guaranteed to be 3000!

Reply:
It's also not guaranteed to be 80--am I missing something?

Reply:
oh sure, but at least we're not explicitly suggesting it is 80 :P saying `:3000` sounds more misleading to me than specifying no port

Reply:
I mean here is the thing: it really doesn't matter what we say here, but here is why I think we shouldn't *change* what we say here: it will create a diff in every single generator's test job, since the port will make it into the generated code (even if it is then overridden by the unit tests at runtime). So since it provides no value to make this change, I'd spare the generator owners from that diff 😉 the https/http is fine I guess, but out of curiosity, on which OS do you have to configure something? We've never seen issues, works out of the box on Windows, Linux, Mac, including build servers like Travis and so on

Reply:
SSL on localhost does not work unless you set up the local server to use it. I would speculate it's working for most people because they're specifying their own base URL. The only reason to make this change this is so that the defaults for the testserver "just work" with the defaults for the generated clients.

It's reasonable to say that the overhead of people having to sift through this diff might be greater than the convenience of not having to specify a base URL when they're using the default server configuration.